### ISSUE TITLE
Remove braces for let expressions

### ISSUE THREAD

**Issue by janmasrovira** [anoma/juvix#1786]
Currently the syntax for let expression is as follows:
```
let {
  stmt1;
  stmt2;
 ...
} in body;
```
Since both `let` and `in` are reserved keywords, the braces are not necessary for parsing, thus, it would make sense to remove the braces.

**Comment by lukaszcz** [comment]
I agree. They serve no purpose, and when you want to write a quick single-variable let the braces add a non-negligible typing cost.
